lighthouse-cli/sentry-prompt.js

```diff
@@ -42,22 +49,32 @@ async function prompt() {
   });
 
   return Promise.race([
-    prompt.then((result: {isErrorReportingEnabled: boolean}) => {
-      clearTimeout(timeout);
+    prompt.then(result => {
+      clearTimeout(/** @type {NodeJS.Timer} */ (timeout));
       return result.isErrorReportingEnabled;
     }),
     timeoutPromise,
   ]);
 }
 
-export async function askPermission() {
+/**
+ * @return {!Promise<boolean>}
```

Comment:
does `inquirer` not have types exposed? it'd be nice to reuse them if possible

Reply:
we actually do use them above in `prompt()`. For here do you just mean so that the return type doesn't have to be explicit? Or something else.

(unfortunately one of the few places the `inquirer` type checking is used is broken. `inquierer.promprt()` returns a promise decorated with an additional `ui` property (allowing `prompt.ui.close()`), but the [type declaration](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/baff013c6e582beff91f727169fa135f32bc017c/types/inquirer/index.d.ts#L35) just calls it a promise, which is why that `// @ts-ignore` line has to be up there)

Reply:
Oh they don't have the proper types haha got it carry on 👍